app/service/utils.go

```diff
@@ -264,9 +264,21 @@ func setConvertedValueToJSONMap(valueName string, value interface{}, result map[
 		value = strconv.FormatInt(valueInt64, 10)
 	}
 
+	value = convertDateToRFC3339(value, snakeCaseName)
 	result[snakeCaseName] = value
 }
 
+func convertDateToRFC3339(value interface{}, snakeCaseName string) interface{} {
```

Comment:
Wouldn't `convertToRFC3339IfDate` be more appropriate as name? From what I understand, you do not only give dates as args.

Reply:
fixed